pkg/sanitizer/helpers.go

```diff
@@ -10,36 +10,61 @@ import (
 
 const blocked = "blocked"
 
-type ClusterMap map[string]string
+// ClusterInfo holds the provider, capacity, and capabilities.
+type ClusterInfo struct {
+	Provider     string
+	Capacity     int
+	Capabilities []string
```

Comment:
Do we maybe want to create a type Capabilty? So we don't compare it with a string value?
Could we maybe add `capabilities` stanza to the config so we know what capabilities are available?
```
capabilities:
   - vpn
   - aarch64
   - arm64
aws:
...
blocked:
...
```

Reply:
Just ideas....

Reply:
string is fine because labels in prowjob will be strings as well